dev/EnvironmentManager/ChangeTrackerHelper/ChangeTrackerHelper/ChangeTrackerHelper.h

```diff
@@ -0,0 +1,56 @@
+﻿#pragma once
+#include <windows.h>
+#include <appmodel.h>
+
+enum AreChangesTracked
+{
+    YES,
+    NO,
+    NOT_CHECKED
+};
+
+AreChangesTracked m_AreChangesTracked{NOT_CHECKED};
```

Comment:
P.S. If you do enums like this it's usually best to make the default/none/Unknown value =0 e.g.

```
enum class AreChangesTracked
{
    Unknown = 0,
    No,
    Yes
}
```

so then folks can easily write

```
AreChangesTracked g_areChangesTracked{};
```

and they're correct even though they didn't specify `Unknown`